build.gradle.kts

```diff
@@ -81,7 +81,7 @@ loom {
                 "-Dfabric-api.datagen.output-dir=${file("src/generated/output")}",
                 "-Dfabric-api.datagen.modid=slotlink_datagen"
             )
-            runDir("${project.buildDir}/datagen")
+            runDir("${project.buildDir.resolve("datagen").relativeTo(project.projectDir)}")
```

Comment:
How exactly does it fail on windows?

Reply:
`${project.buildDir}` is replaced by the absolute path, and the parameter to runDir is appended to the projectDir absolute path, repeating the directory prefix. It gives something like `C:\Path\To\slotlink\C:\Path\To\slotlink\run\datagen`.

Reply:
Right now I realize this might be more an error in the loom plugin, that accepts only String where it should accept File and maybe Path, than an issue the configuration. (And BTW, the whole string template should go away)

Reply:
That stinks. Well, I would prefer it to just be `build/datagen` then.

Reply:
I have removed the commit from this PR, since it has nothing to do with the translation.